tests/ceph_ansible/switch_rpm_to_container.py

```diff
@@ -10,12 +12,57 @@ def run(**kw):
     ceph_nodes = kw.get("ceph_nodes")
     config = kw.get("config")
     build = config.get("rhbuild")
+    docker_registry = config.get("ceph_docker_registry")
+    docker_image = config.get("ceph_docker_image")
+    docker_image_tag = config.get("ceph_docker_image_tag")
     installer_node = None
     ansible_dir = "/usr/share/ceph-ansible"
     playbook = "switch-from-non-containerized-to-containerized-ceph-daemons.yml"
     for cnode in ceph_nodes:
         if cnode.role == "installer":
             installer_node = cnode
+    log.info("Get all.yml content")
+    out, err = installer_node.exec_command(
+        sudo=True,
+        cmd="cat {ansible_dir}/group_vars/all.yml".format(ansible_dir=ansible_dir),
+    )
+    conf = yaml.safe_load(out)
+    log.info("Previous all.yml file content : \n %s" % conf)
+
+    if conf.get("ceph_repository_type") == "cdn":
+        out, err = installer_node.exec_command(
+            sudo=True,
+            cmd="cat {ansible_dir}/group_vars/all.yml.sample".format(
+                ansible_dir=ansible_dir
+            ),
+        )
+        sample_conf = yaml.safe_load(out)
+        ceph_docker_registry = sample_conf.get("ceph_docker_registry")
+        ceph_docker_image = sample_conf.get("ceph_docker_image")
+        ceph_docker_image_tag = sample_conf.get("ceph_docker_image_tag")
+    else:
```

Comment:
I think we don't need `else` part, since dev conatiner image values are already set at the beginning.